pkg/gateway/playback_test.go

```diff
@@ -128,7 +140,16 @@ func getBasicHandler(t *testing.T, authService *simulator.PlayBackMockConf) (htt
 		DB:     conn,
 	})
 	testutil.MustDo(t, "build c", err)
-	multipartsTracker := multiparts.NewTracker(conn)
+
+	var multipartsTracker multiparts.Tracker
+	dbParams := conf.GetDatabaseParams()
```

Comment:
I don't think the test code need to use the configuration - it should setup the right services composition it would like to test

Reply:
Removed playback test